### ISSUE TITLE
Exception thrown is not reported/captured

### ISSUE THREAD

**Issue by ewized** [eclipsesource/J2V8#296]
When a method from Java throws an exception called from the javascript object, I can not seem to capture the exception (the exception is not printing any stack trace either). The only way I can capture the exception is to put a try/catch in the method that will throw the exception.

```
public void register() {
    V8Object object = new V8Object(engine);
    for (Method method : getClass().getMethods()) {
        if (method.getAnnotation(Bind.class) != null) {
            object.registerJavaMethod(this, method.getName(), method.getName(), method.getParameterTypes()); // Would be nice if I can attach some type of ExceptionHandler
        }
    }
}

@Bind public void method() { // if I put try/catch here I can capture the exception
    if (true) throw new RunetimeException();
}
```

**Comment by irbull** [comment]
Can you catch the exception in Javascript? It was designed that it should
propagate to a Javascript exception.

Having said that, it is sometimes clunky to catch and handle Java
exceptions in JS. I really like the idea of attaching an exception handler
when the method is registered.

**Comment by ewized** [comment]
I see, the string of the message is thrown in the JS side, but I did not handle the exception. I also did not get any message to know the exception was thrown and uncaught.

**Comment by irbull** [comment]
Yes, we throw it as a string in JS. I wasn't really sure what else to do.
If you have some suggestions for the JS side, feel free to open a bug. As
for the Java side, I really like this proposal.

**Comment by adam-law** [comment]
Any progress here? i.e. Java-side exception handling
